framework/auth/cas.py

```diff
@@ -99,7 +100,7 @@ def service_validate(self, ticket, service_url):
         url.args['ticket'] = ticket
         url.args['service'] = service_url
 
-        resp = requests.get(url.url)
+        resp = requests.get(url.url, verify=False)
```

Comment:
this is a left over from shibboleth-cas institution login local development, not sure how this ends up here, probably from stash, should be removed